src/pages/Home/NewUserPromo/NewUserPromo.js

```diff
@@ -6,9 +6,9 @@ import { HeadingMedium, BodyLarge } from 'src/components/Typography';
 import { dataLargeCSS } from 'src/components/Typography/Data';
 import Link from 'src/components/Link';
 
-import { ReactComponent as BorrowsETHIcon } from 'src/assets/images/splash/borrow-sETH.svg';
-import { ReactComponent as BuysETHIcon } from 'src/assets/images/splash/buy-sETH.svg';
-import { ReactComponent as BuysUSDIcon } from 'src/assets/images/splash/buy-sUSD.svg';
+import { ReactComponent as BorrowsETHIcon } from 'src/assets/images/splash/borrow-seth.svg';
+import { ReactComponent as BuysETHIcon } from 'src/assets/images/splash/buy-seth.svg';
+import { ReactComponent as BuysUSDIcon } from 'src/assets/images/splash/buy-susd.svg';
```

Comment:
why is this needed?

Reply:
It breaks my linter. My file system says they're lowercase. Isn't it the same for you?